data-prepper-plugins/sqs-source/build.gradle

```diff
@@ -0,0 +1,14 @@
+dependencies {
+    implementation project(':data-prepper-api')
+    implementation project(':data-prepper-plugins:aws-plugin-api')
```

Comment:
I don't think you need the `aws-plugin-api`

Reply:
Thanks @asifsmohammed  for your review, It is required because i am using AwsCredentialsOptions from aws-plugin-api.
Removed **aws-plugin** dependency because its not used in current PR#2787